mp4parse/src/lib.rs

```diff
@@ -267,6 +267,8 @@ pub enum AudioCodecSpecific {
     ES_Descriptor(ES_Descriptor),
     FLACSpecificBox(FLACSpecificBox),
     OpusSpecificBox(OpusSpecificBox),
+    AC3SpecificBox(Vec<u8>),
+    EC3SpecificBox(Vec<u8>),
```

Comment:
Unless we plan to support these codecs, it seems like a waste of time reading the box contents and storing them just to ignore them later.  Skipping over them would also simplify the parsing code slightly.  Thoughts?